App.js

```diff
@@ -1,18 +1,150 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { Alert, AppRegistry, Button, StyleSheet, Text, TextInput, View } from 'react-native';
+import { StackNavigator } from 'react-navigation';
+import Picker from 'react-native-picker';
+
+// export default class LogIn extends React.Component {
+//   // static navigationOptions = {
+//   //   title: 'Sign Up',
+//   // };
+//   constructor(props) {
+//     super(props);
+//     this.state = {text: ''};
+//   }
+//
+//   render() {
+//     return (
+//       <View>
+//         <View style={{padding: 10}}>
+//           <TextInput
+//             style={{height: 40}}
+//             placeholder="Username"
+//           />
+//           <Text style={{padding: 10, fontSize: 42}}>
+//           </Text>
+//         </View>
+//         <View style={{padding: 10}}>
+//           <TextInput
+//             style={{height: 40}}
+//             placeholder="Password"
+//           />
+//           <Text style={{padding: 10, fontSize: 42}}>
+//           </Text>
+//         </View>
+//         <Button
+//           onPress={() => { Alert.alert('Account Created!')}}
+//           title="LOG IN"
+//         />8
+//       </View>
+//     );
+//   }
+// }
+
+// export default class SignUp extends React.Component {
+//   // static navigationOptions = {
+//   //   title: 'Sign Up',
+//   // };
+//   constructor(props) {
+//     super(props);
+//     this.state = {text: ''};
+//   }
+//
+//   render() {
+//     return (
+//       <View>
+//         <View style={{padding: 10}}>
+//           <TextInput
+//             style={{height: 40}}
+//             placeholder="First Name"
+//           />
+//           <Text style={{padding: 10, fontSize: 42}}>
+//           </Text>
+//         </View>
+//         <View style={{padding: 10}}>
+//           <TextInput
+//             style={{height: 40}}
+//             placeholder="Last Name"
+//           />
+//           <Text style={{padding: 10, fontSize: 42}}>
+//           </Text>
+//         </View>
+//         <View style={{padding: 10}}>
+//           <TextInput
+//             style={{height: 40}}
+//             placeholder="Email"
+//           />
+//           <Text style={{padding: 10, fontSize: 42}}>
+//           </Text>
+//         </View>
+//         <View style={{padding: 10}}>
```

Comment:
check eslint- this would be failing your linter check. Object spacing (should have a space before/after the curly braces on the inside. e.g. `{{ padding: 10 }}`.